### PR TITLE
Add autoplay feature, resolve issue52

### PR DESCRIPTION
**Pull request by cocolin041**
Co-authored-by: Coco Lin mathstatloversihyilin@gmail.com
Co-authored-by: Lyu Ji lyuji220@hotmail.com

This pull request solve the issue52. We added the feature of `checkbox` to allow users choose whether to autoplay the video, with the default set to be `true`. 
This is now working on safari, but not on Chrome, so we will create a new issue about how to make this feature working on all browsers.

### CODE REVIEW

**Comment by claysquareninety** [comment]
Sending you a screenshot. It looks like adding this caused the controls to get a bit messy. I can't use the volume slider and the play button is nowhere to be found.

File: src/views/Receiver.vue [hunk 1]
```diff
@@ -67,21 +67,28 @@ video
 #media-controls
   width: 60%
   animation: fade-in 0.4s
+  > *
```
Comment (claysquareninety) [inline src/views/Receiver.vue]: Instead of using * here can you use specific classes to make it more clear?

File: src/views/Receiver.vue [hunk 2]
```diff
@@ -67,21 +67,28 @@ video
...
+    max-width: 120px
+    @supports (-webkit-touch-callout: none) // iOS volume slider doesn't work, so hide it
+      visibility: hidden
 
-#volume-slider
-  max-width: 120px
-  @supports (-webkit-touch-callout: none) // iOS volume slider doesn't work, so hide it
-    visibility: hidden
+  #autoplay
+    > *
```
Comment (claysquareninety) [inline src/views/Receiver.vue]: here too

File: src/views/Receiver.vue (hunk 1 above)
Comment (cocolin041) [reply src/views/Receiver.vue]: okay! Fixed!

**Pull request merged**